data/test-stable-addresses-cu2.c

```diff
@@ -0,0 +1,10 @@
+extern unsigned int factorial(unsigned int n);
+
+static void
```

Comment:
compiler might inline this, so best to add `__attribute__((noinline))` if the test expects this function to be preserved (might still be not called from `foo` even with that, btw, you'd need `asm volatile ("")` inside `factorial_wrapper` to prevent compiler from inlining the logic of factorial_wrapper inside `foo`; static functions give a lot of freed to compiler)

Reply:
Yeah, we can tag it as `noinline`. We deliberately compile without optimizations, so we haven't gotten screwed.